React/Lesson41/redux-app/src/Product/Product.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FaShoppingCart } from 'react-icons/fa';
+import { incrementClickCount, incrementCartItemCount } from '../redux/actions';
+import 'bootstrap/dist/css/bootstrap.min.css';
```

Comment:
you have to import in `main.jsx` for whole application
not just for 1 component